src/main/java/school/lemon/changerequest/java/introduction/hw2/Task2.java

```diff
@@ -3,6 +3,19 @@
 
 public class Task2 {
     public static String firstTwo(String s) {
-        return "";
+
+        if (s == null)
+            return null;
+        StringBuilder shortstr = new StringBuilder(s);
+        if (s.length() < 2)
+            return s;
+
+
+        else {
+            shortstr.setLength(2);
+            String substring = shortstr.substring(0);
+            return substring;
+        }
+
```

Comment:
it means like this ?https://github.com/ChangeRequest/introduction.hw2/pull/3/commits/6cac7fce03467948a827e1be8025214e1323158c